controller.go

```diff
@@ -181,6 +258,47 @@ func (c *controller) NewNetwork(networkType, name string, options ...NetworkOpti
 	return network, nil
 }
 
+func (c *controller) newNetworkFromStore(n *network) {
+	c.Lock()
+	n.ctrlr = c
+	n.driver = c.drivers[n.networkType]
```

Comment:
This will panic if no driver exists for the  `networkType`. There are a few this that needs to happen here:
* We need to handle this gracefully
* We should not add the network into our local table if the driver doesn't exist
* We need to call `driver.CreateNetwork` when this happens.